src/oatpp/web/mime/multipart/PartList.hpp

```diff
@@ -74,12 +75,21 @@ class PartList : public Multipart {
 
   /**
    * Get part by name <br>
+   * Returns the first part if multiple parts with same name exist.
    * Applicable to named parts only.
    * @param name - &id:oatpp::String;.
    * @return - &id:oatpp::web::mime::multipart::Part;.
    */
   std::shared_ptr<Part> getNamedPart(const oatpp::String& name);
 
+  /**
+   * Get all parts by name <br>
+   * Applicable to named parts only.
+   * @param name - &id:oatpp::String;.
+   * @return - std::list<std::shared_ptr<&id:oatpp::web::mime::multipart::Part;>>.
```

Comment:
please escape triangle brackets:

```
@return - std::list<std::shared_ptr<&id:oatpp::web::mime::multipart::Part;>>.
```

change to

```
@return - std::list&lt;std::shared_ptr&lt;&id:oatpp::web::mime::multipart::Part;&gt;&gt;.
```

Reply:
I used the style of `getAllParts`